src/prefect/backend/__init__.py

```diff
@@ -0,0 +1 @@
+import prefect.backend.kv_store
```

Comment:
It seems reasonable to expose these top-level here as

`from prefect.backend import set_key_value` 

but I'm not sure if I feel strongly

Reply:
done